client/src/layers/UILayers/ToolTipWidget.vue

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+import {
+  defineComponent, PropType, Ref,
+} from '@vue/composition-api';
+import { StateStyles } from 'vue-media-annotator/use/useStyling';
+
+export default defineComponent({
+  name: 'ToolTipWidget',
+  props: {
+    color: {
+      type: Function as PropType<(name: string) => string>,
+      required: true,
+    },
+    stateStyling: {
+      type: Object as PropType<StateStyles>,
+      required: true,
+    },
+    textSettings: {
+      type: Function as PropType<(name: string) => { showLabel: boolean; showConfidence: boolean }>,
+      default: () => ({ showLabel: true, showConfidence: true }),
+    },
+    dataList: {
+      type: Object as PropType<Ref<[string, number, number][]>>,
+      default: () => [],
+    },
```

Comment:
This isn't a descriptive tuple.  Could it be an object instead?  `{ x, y, type }` or something like that?

If you don't want to change it, can you document what it is here?